modules/UI.js

```diff
@@ -0,0 +1,74 @@
+export default class Book {
```

Comment:
- [ ] Please use the ES6 features as required for this milestone, arrow functions instead of class methods! 🙏

Reply:
I have done as much as I can sur